src/components/Portfolio/LoanStatsTable.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+	<div class="tw-flex tw-justify-between tw-mb-2 tw-bg-[#efefef] tw-p-2">
```

Comment:
#efefef is a named color, $platinum in: src/assets/scss/_settings.scss

Reply:
Okay...  I just went with tw-bg-gray-200 instead so it can be done within tailwind instead of css, if thats ok.